app/controllers/pets_controller.rb

```diff
@@ -35,4 +42,10 @@ def pet_params
     params.permit(:image, :name, :description, :age, :sex)
   end
 
+  def missing(parameters)
+    parameters.keys.find_all do |key|
+      (key != "image" && key != "description" ) && params[key].empty?
+    end
+  end
```

Comment:
image and description fields are not required and so will not trigger the flash message